docs/conf_works.py

```diff
@@ -0,0 +1,348 @@
+# -*- coding: utf-8 -*-
```

Comment:
What is the purpose of this file? Is it needed and how is it triggered?
Certainly the copyright seems a bit off :-/.

Reply:
Sorry, I was testing two different conf.py files. I'll delete this and ignore the ds_store file

Reply:
Thanks :+1:.